### ISSUE TITLE
Missing Lap and Pause information in .slf file

### ISSUE THREAD

**Issue by Ottomo1975** [mrihtar/Garmin-FIT#5]
@mrihtar , translation works now perfect but some information is missing in the .slf file.
The information about laps and break times are not visible in the Data Center.
Would that something be you could implement in your script?

**Comment by mrihtar** [comment]
Laps and break times (= markers in .slf) are missing because I didn't have/find any .slf example with markers and I didn't know the complete format (of .slf, not .fit). Now I have at least one (yours). It would be good if you can send me (privately) some more examples. The one you sent seems to have only one "lap", all others are "pause", so it would be good if the examples include more "laps".

**Comment by Ottomo1975** [comment]
@mrihtar, I did an exercise yesterday and created a .slf file with some laps included.
How could I send that file to you privatly?

**Comment by mrihtar** [comment]
Hi @Ottomo1975! You can send the file to my e-mail, which you can find via one of the following methods: [Sourcecon](https://www.eremedia.com/sourcecon/how-to-find-almost-any-github-users-email-address/) or [Stackoverflow](http://stackoverflow.com/questions/12686545/how-to-leave-a-message-for-github-com-user/12687679#12687679). Sorry for this complication, it would be much faster if Github hasn't removed private messaging.

**Comment by sergei1998** [comment]
Hi mrihtar!    I need your help.  Please give your vayber or e-mail!   Write me an e-mail   sergei1998@mail.ru.

**Comment by vicgor** [comment]
[.FIT file with pauses](https://dl.dropboxusercontent.com/u/8996908/3.fit) converted from .SLF
 [.SLF source](https://dl.dropboxusercontent.com/u/8996908/2016_11_01__11_14.slf)

**Comment by mpnegro** [comment]
Hi,
I've the same problem, and the distance show is wrong. If you wana I can send you some files.
Regards

**Comment by mrihtar** [comment]
Yes please, if you can, send me some example files, so that I can compare the output from different devices.

**Comment by mpnegro** [comment]
Let me know the "address" to send.

**Comment by mrihtar** [comment]
Would you be so kind and follow my advice to Ottomo1975 from above how to find out an e-mail of a Github contributer (via [Sourcecon](https://www.eremedia.com/sourcecon/how-to-find-almost-any-github-users-email-address/) or [Stackoverflow](http://stackoverflow.com/questions/12686545/how-to-leave-a-message-for-github-com-user/12687679#12687679)).

**Comment by mpnegro** [comment]
Send you the files
